packages/react/slot/src/Slot.tsx

```diff
@@ -101,18 +104,6 @@ function mergeProps(slotProps: AnyProps, childProps: AnyProps) {
   return { ...slotProps, ...overrideProps };
 }
 
-type EventHandler = (...args: unknown[]) => unknown;
-
-function composeHandlers(childHandler?: EventHandler, slotHandler?: EventHandler) {
-  return function handleEvent(...args) {
-    childHandler?.(...args);
-    const isDefaultPreventedEvent = args[0] instanceof Event && args[0].defaultPrevented;
```

Comment:
This was always returning `false` anyway because it's a SyntheticEvents so `args[0] instance of Event` was always `false`.